cms/datasets/tests/test_models.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+import responses
+from django.test import TestCase
+
+from cms.datasets.models import DATASETS_BASE_API_URL, ONSDataset, ONSDatasetApiQuerySet
+
+
+class TestONSDatasetApiQuerySet(TestCase):
+    @responses.activate
+    def test_count_uses_total_count(self):
+        responses.add(responses.GET, DATASETS_BASE_API_URL, body=json.dumps({"total_count": 2, "items": []}))
```

Comment:
Suggestion: Rather than manually encoding the JSON data for the body, you can pass the `dict` directly to `json=`